product_docs/docs/epas/15/epas_rel_notes/epas15_2_0_rel_notes.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Version 15.2.0"
+---
+
+EDB Postgres Advanced Server 15.2.0 includes the following enhancements and bug fixes:
+
+| Type           | Description                                                                                                                          |  Category              |
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------------|  --------------------- |
+| Upstream merge | Merged with community PostgreSQL 15.2. See the [PostgreSQL 15 Release Notes](https://www.postgresql.org/docs/15/release-15.html) for more information.                           |                        |
+| Feature        | Transparent Data Encryption (TDE) - TDE is an optional feature supported by EDB Postgres Advanced Server with high availability. It encrypts, transparently to the user, any user data stored in the database system. User data includes the actual data stored in tables and other objects, as well as system catalog data such as the names of objects. See [TDE docs](/tde/latest) for more information. | Security |
+| Enhancement    | EDB Postgres Advanced Server now allows non-superuser to load data using EDB*Loader.     | edbldr |                       
+| Enhancement    | Enable multi-insert support dynamic partition for EDB*Loader and COPY command. |  |
```

Comment:
I can't make sense of this one. Can you get someone to explain what this is supposed to be saying? At minimum, I think "support" is misplaced.

Reply:
done